modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/RpcRequests.java

```diff
@@ -48,6 +48,11 @@ private RpcRequests() {
          * <code>optional string errorMsg = 2;</code>
          */
         String errorMsg();
+
+        /**
+         * <code>optional string leaderId = 3;</code>
+         */
+        String leaderId();
```

Comment:
Within the header of the file we have following warning
```
// Generated by the protocol buffer compiler.  DO NOT EDIT!
// source: rpc.proto
```
Does it mean that given file is generated and shouldn't be edit?

inappropriate javadoc format.

Reply:
@sanpwc Honestly, have no answer, but we changed this file many times already :)

Reply:
Update by nullable annotation and change javadocs (looks like it is ok to change this file already)